data-structures-linked-lists/append.js

```diff
@@ -0,0 +1,11 @@
+/* global LinkedList */ // eslint-disable-line no-unused-vars
+/* exported append */
+
+function append(list, value) {
+  const newList = new LinkedList(value);
+  let listCopy = list;
```

Comment:
Careful, this is not a copy. Fortunately we don't need a copy, we are _totally_ supposed to mutate this data structure.